pandas/core/resample.py

```diff
@@ -1269,11 +1267,9 @@ def _downsample(self, how, **kwargs):
         """
         how = com.get_cython_func(how) or how
         ax = self.ax
-        if self._selected_obj.ndim == 1:
-            obj = self._selected_obj
-        else:
-            # Excludes `on` column when provided
-            obj = self._obj_with_exclusions
+
+        # Excludes `on` column when provided
+        obj = self._obj_with_exclusions
```

Comment:
I'm a bit weak on this area, but why is this the same as the old code?

Reply:
bc in the ndim==1 case _selected_obj and _obj_with_exclusions are the same xref #46944

Reply:
👍